examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -3407,7 +3384,7 @@ server cluster ThreadNetworkDiagnostics = 53 {
 
   struct SecurityPolicy {
     INT16U rotationTime = 0;
-    BITMAP16 flags = 1;
+    bitmap16 flags = 1;
```

Comment:
this is inconsistent - I see that single/double/enum became uppercase ... but why did bitmap become lowercase?

Reply:
Mentioned above. Should I just change everything to uppercase or lowercase?